src/config/dynamic-theme-fixes.config

```diff
@@ -3,6 +3,7 @@
 INVERT
 .jfk-bubble.gtx-bubble
 .captcheck_answer_label > input + img
+span#closed_text > img[src="https://www.gstatic.com/images/branding/googlelogo/2x/googlelogo_dark_color_84x28dp.png"]
```

Comment:
Not sure if the sizes changes depending on the screen resolution might change it into
```CSS
span#closed_text > img[src^="https://www.gstatic.com/images/branding/googlelogo"]
```